codegen/src/test/resources/software/amazon/awssdk/codegen/poet/common/test-enum-class.java

```diff
@@ -37,21 +22,20 @@ private TestEnumClass(String value) {
 
     @Override
     public String toString() {
-        return value;
+        return String.valueOf(value);
     }
 
     /**
-     * Use this in place of valueOf.
+     * Use this in place of valueOf to convert the raw string returned by the service into the enum value.
      *
      * @param value
      *        real value
      * @return TestEnumClass corresponding to the value
      */
     public static TestEnumClass fromValue(String value) {
-        if (isBlank(value)) {
-            throw new IllegalArgumentException("Value cannot be null or empty!");
+        if (value == null) {
```

Comment:
Why the change in behavior here?

Reply:
It was easier to use this way. If the service returned null, you'd have to do a null and isEmpty check before converting it to an enum. This way, a null response from the service is a null enum, and an empty-string response from the service is an UNKNOWN enum.